frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx

```diff
@@ -104,7 +104,7 @@ export function NotebookSelectList(props: NotebookSelectProps): JSX.Element {
 
     return (
         <div className="space-y-2 flex flex-col flex-1 h-full overflow-hidden">
-            <div className="border-b space-y-2 flex-0">
+            <div className="border-b space-y-2 flex-0 pb-2">
```

Comment:
We have space-y-2 and pb-2 conflicting with each other. Fine if its necessary just something that normally makes me think, one of the values needs deleting and the other changing